cpp/cmake/thirdparty/CUDF_GetRMM.cmake

```diff
@@ -0,0 +1,41 @@
+#=============================================================================
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(find_and_configure_rmm VERSION)
+
+    if(ARGC GREATER 2 AND (ARGV1 AND ARGV2))
+        set(BUILD_TESTS OFF)
+        set(BUILD_BENCHMARKS OFF)
+        add_subdirectory("${ARGV1}" "${ARGV2}" EXCLUDE_FROM_ALL)
+        return()
+    endif()
+
+    CPMFindPackage(NAME rmm
+        VERSION         ${VERSION}
+        GIT_REPOSITORY  https://github.com/rapidsai/rmm.git
+        GIT_TAG         branch-${VERSION}
```

Comment:
Is there ever a situation where the RMM version and cudf version would mismatch? I.E. cudf gets hotfixed and rmm doesn't?

Also, for hotfixes, I believe they're only pushed to `master` and not `branch-xxx`. How would someone build the current release?

Reply:
> Is there ever a situation where the RMM version and cudf version would mismatch?

I don't know, maybe?

> I.E. cudf gets hotfixed and rmm doesn't?

CUDF hotfixes should be fine since we're using `major.minor` for the RMM version, and presumably hotfix versions would be reflected in the patch version. RMM hotfixes would require changing something here.

Reply:
Ops team is discussing this and coming back with a branching / tagging strategy that should resolve this.

Reply:
This is resolved now!